### ISSUE TITLE
tvOS support

### ISSUE THREAD

**Issue by iliaskarim** [LoganWright/Genome#15]
`import Genome` currently results in the following build error on Xcode 7.1 beta 3:
```
module file was created for incompatible target x86_64-apple-ios8.0: [...]/Carthage/Build/iOS/Genome.framework/Modules/Genome.swiftmodule/x86_64.swiftmodule
```
It would be great to see this project support tvOS like Alamofire and Decodable have!

**Comment by LoganWright** [comment]
I'm going to leave this open until tvOS and Xcode 7.1 are officially released.  For those who end up here, point your Cartfile to the `<a href="https://github.com/LoganWright/Genome/tree/xcode7.1">xcode7.1</a> branch.

**Comment by LoganWright** [comment]
@iliaskarim I just made some small changes to the Xcode7.1 branch adding some of the new xcode files to the gitignore.  Let me know if you have problems.  Seemed to run fine for me.

Also, thanks for your contributions!

**Comment by iliaskarim** [comment]
Excellent--adding .xcsmblueprint to the gitignore like you did seems good to me

**Comment by LoganWright** [comment]
Fixed in #23
